### ISSUE TITLE
Fix A11Y violation in card basic

### ISSUE THREAD

**Issue by Elohina** [buildit/gravity-ui-web#375]
**Describe the bug**
The `card-basic` component is generating an accessibility violation since `aria-labelledby` is not being referenced correctly.

I think this can be solved by working in the `inline-svg-symbol` component because what it is causing the error is the `__title` added at the end of the `aria-labelledby` property.

**To Reproduce**
1. Go to /gravity-ui-nunjucks.
2. Run npm run test:a11y.
3. See the `card-basic` violation in the list of a11y violations.

**Expected behavior**
```
 Ensures all ARIA attributes have valid values
 ARIA attributes must conform to valid values
 Help: https://dequeuniversity.com/rules/axe/3.4/aria-valid-attr-value?application=axe-puppeteer
```
